.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,66 @@
+name: Bug Report
+description: Report a problem in Neovim
+labels: [bug]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Before reporting: search [existing issues](https://github.com/neovim/neovim/issues?q=is%3Aissue+is%3Aopen+label%3Abug) and check the [FAQ](https://github.com/neovim/neovim/wiki/FAQ).
+
+  - type: input
+    attributes:
+      label: "Neovim Version"
+      description: "`nvim --version`:"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: "Operating system/version:"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: "Terminal name/version:"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: "TERM environment variable"
+      description: "echo `$TERM`:"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: "Installation"
+      description: "How did you install neovim: build from repo / system package manager / appimage / homebrew / snap / chocolatey / other (please specify)?"
+      placeholder: "Arch User Repository (AUR)"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: "Reproduce"
```

Comment:
Maybe the old "Steps to reproduce" would be clearer.

Reply:
Did you mean that in the "label" field  I should replace the string "Reproduce" with "Steps to reproduce"? Or did you mean I should change back the whole section like it was previously?

Reply:
The first: just the label.

Reply:
fixed